### DISCUSSION TITLE
Design System documentation

### DISCUSSION THREAD

**Discussion by mojtabamoosavi** [fuma-nama/fumadocs#991] [Q&A]
I want to use Fumadocs for documenting a design system, but I'm having issues with the component demo. 
The problem is that Fumadocs' own styles are affecting the components.

Our design system is not built with Tailwind; instead, we used Pandacss. I also created the Fumadocs project without Tailwind and added the styles to the project by importing ```fumadocs-ui/style.css```.

@fuma-nama

**Comment by fuma-nama** [comment]
It should not affect them except a default border color was added. What styles are conflicting?

**Reply by mojtabamoosavi** [reply]
Thank you for your response.

For example, the Button component: 
These styles are causing an issue.

## With these styles
Styles:
![image](https://github.com/user-attachments/assets/beae45b3-ee5d-40b4-8373-fa260ffe13a7)

Result:
![image](https://github.com/user-attachments/assets/f3f4cae0-e703-4831-a12f-1d8618d664f7)

## Without these styles
Styles:
![image](https://github.com/user-attachments/assets/23d3fdcd-8d6c-4f6c-8eee-557a75a9f2e2)

Result:
![image](https://github.com/user-attachments/assets/aa1ff87b-d0a0-4520-8909-0e09acd3e314)

**Reply by fuma-nama** [reply]
These are CSS resets applied by Tailwind CSS so it looks consistent across all browsers, make sure your panda CSS styles are not overwritten by it and you don’t rely on default browser styles
